hwilib/commands.py

```diff
@@ -178,9 +179,13 @@ def getkeypool(args, client):
     return import_data
 
 def displayaddress(args, client):
-    if args.sh_wpkh == True and args.wpkh == True:
-        return json.dumps({'error':'Both `--wpkh` and `--sh_wpkh` can not be selected at the same time.','code':BAD_ARGUMENT})
-    return client.display_address(args.path, args.sh_wpkh, args.wpkh)
+    if args.desc != None:
+      key = Descriptor.parse(args.desc, args.testnet)
+      return client.display_address(key.path, key.sh_wpkh, key.wpkh)
+    elif args.path != None:
+      return client.display_address(args.path, args.sh_wpkh, args.wpkh)
+    else:
+      return {'error':'Either --descriptor or --path must be set','code':BAD_ARGUMENT}
```

Comment:
The option is `--desc`, not `--descriptor`.

Using a mutually exclusive group removes the need for this error.

Reply:
Fixed typo, but if I leave out the check the command just returns `null`:
`./hwi.py --testnet --fingerprint 00000000 displayaddress`